### PR TITLE
Theme Discovery: Adjust space before first carousel

### PR DESCRIPTION
**Pull request by dsas**
The logged-out theme showcase had more margin between the main header and the first carousel.

## Proposed Changes
- Reduced the spacing between the header component and the first carousel.

## Testing Instructions
 1. Calypso live link
 2. Go to /themes while logged out
 3. The spacing between the header and the first carousel should be reduced.

### Before
Captura de pantalla 2023-10-13 a las 8 53 30

### After
Captura de pantalla 2023-10-13 a las 8 53 15

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/my-sites/themes/theme-showcase.scss [hunk 1]
```diff
@@ -69,6 +69,11 @@
 				padding: 0 32px;
 			}
 		}
+		.themes__showcase {
+			max-width: 1462px;
```
Comment (dsas) [inline client/my-sites/themes/theme-showcase.scss]: Probably there's an appropriate variable somewhere

**Comment by rcrdortiz** [comment]
It looks a bit strange to have all that empty space. Could we make the component behind the feature flag use a different class name so that you all can develop with the fixed width and not have an impact on the production component until we're ready to launch the Discovery project?

**Comment by dsas** [comment]
Well, even if we flag it, once we remove the flag it is still going to look strange, right? Maybe we should do something different..

**Comment by rcrdortiz** [comment]
> Well, even if we flag it, once we remove the flag it is still going to look strange, right? Maybe we should do something different..

I'm adding a link to the slack conversation where we speak about how we're going to move forward with this for future reference:
p1696506023325799/1696347257.368189-slack-C04DZ8M0GHW

**Comment by rcrdortiz** [comment]
I'm taking over this issue. The issue is ready for review.

**Comment by BogdanUngureanu** [comment]
The thumbnails indeed look smaller, however, I think we should also change the size of the placeholders :)


https://github.com/Automattic/wp-calypso/assets/995955/44831587-dfc5-4586-b874-6533f775a06b

**Comment by rcrdortiz** [comment]
> The thumbnails indeed look smaller, however, I think we should also change the size of the placeholders :)
> 
>  Screen.Recording.2023-10-10.at.15.32.13.mov

You need to enable the discovery flag, you're just looking at production from what I can tell from that video you've recorded. I'll update the testing instructions too with the flag for enabling it.

**Comment by rcrdortiz** [comment]
> The thumbnails indeed look smaller, however, I think we should also change the size of the placeholders :)
> 
>  Screen.Recording.2023-10-10.at.15.32.13.mov

Hmm, I'm not sure if we want to have 16:9 aspect ratio now on the production Theme Showcase, reverting to 4:3 aspect ratio.

**Comment by Copons** [comment]
I'm not sure what's going on here. 😄 

The current changes really look the same as before, just with a bit more of top margin.
```css
.themes__showcase {
	margin: 0 auto;
	margin-top: 1em;
}
```

**Comment by rcrdortiz** [comment]
> I'm not sure what's going on here. 😄
> 
> The current changes really look the same as before, just with a bit more of top margin.
> 
> ```css
> .themes__showcase {
> 	margin: 0 auto;
> 	margin-top: 1em;
> }
> ```

You can see in the slack conversation linked in one of the previous comments that we are going to go with three items spanning the full width (so it's just as before these changes).

We can discard this PR if you want, but IMHO the style changes make the component look better.

**Comment by Copons** [comment]
> We can discard this PR if you want, but IMHO the style changes make the component look better.

@rcrdortiz that's cool, but then we need to update all the PR details (title, description, before/after screens). 🙂

**Comment by rcrdortiz** [comment]
> > We can discard this PR if you want, but IMHO the style changes make the component look better.
> 
> @rcrdortiz that's cool, but then we need to update all the PR details (title, description, before/after screens). 🙂

Sure, I've updated the information.

**Review by BogdanUngureanu — APPROVED** [review]
LGTM! :)

**Pull request merged**